package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@maplibre/maplibre-react-native",
   "description": "A MapLibre GL Native plugin for creating maps in React Native",
-  "version": "10.0-alpha.0",
+  "version": "10.0-alpha.1",
```

Comment:
Hey @sarahsporck,

Can we make this `10.0.0-alpha.1` to be semver-compatible?

Reply:
Should be semver compatible already: https://semver.org/#spec-item-9
Or do you prefer a different scheme - lot's to choose from.